homeassistant/components/gdacs/strings.json

```diff
@@ -3,9 +3,13 @@
     "step": {
       "user": {
         "title": "Fill in your filter details.",
-        "data": { "radius": "Radius" }
+        "data": {
+          "radius": "Radius"
+        }
       }
     },
-    "abort": { "already_configured": "Location is already configured." }
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
```

Comment:
I think we should use `already_configured_location` here https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json#L60